lang/english.xml

```diff
@@ -492,8 +502,8 @@
 		<Replace Tag="LOC_ABILITY_HUSZAR_DESCRIPTION" Language="en_US">
 			<Text>+2 [ICON_Strength] Combat Strength from every City-State of which you are suzerain.</Text>
 		</Replace>
-		<Row Tag="LOC_ABILITY_HUSZAIR_COMBAT_PREVIEW" Language="en_US">
-			<Text>From:</Text>
+		<Row Tag="LOC_ABILITY_HUSZAIR_COMBAT_PREVIEW" Language="en_US">	<!-- Change this word so it fits your language syntax and style! (for example: +2 City-State Armagh) -->
+			<Text>from</Text>
```

Comment:
Did you mean to change this text?  Just making sure you didn't do a find and replace.